src/IdeaStatiCa.BimImporter/Importers/CutImporter.cs

```diff
@@ -27,7 +27,20 @@ protected override object ImportInternal(IImportContext ctx, IIdeaCut cut, Conne
 					PlanePoint = ctx.Import(workPlane.Origin).Element as Point3D,
 
 				};
-				var beamIOM = connectionData.Beams.Find(b => b.OriginalModelId == cut.ModifiedObject.Id);
+				var beamIOM = connectionData.Beams.Find(b =>
```

Comment:
šla by udělat funkce, jestli se to stejné volá níže?

Reply:
fixed